compiler/aarch64/codegen/ARM64Instruction.hpp

```diff
@@ -2866,6 +2866,81 @@ class ARM64Trg1Src2ExtendedInstruction : public ARM64Trg1Src2Instruction
    virtual uint8_t *generateBinaryEncoding();
    };
 
+class ARM64Trg1Src2IndexedElementInstruction : public ARM64Trg1Src2Instruction
+   {
+   uint32_t _index;
+
+   public:
+
+   /*
+    * @brief Constructor
+    * @param[in] op : instruction opcode
+    * @param[in] node : node
+    * @param[in] treg : target register
+    * @param[in] s1reg : source register 1
+    * @param[in] s2reg : source register 2
+    * @param[in] index : index
+    * @param[in] cg : CodeGenerator
+    */
+   ARM64Trg1Src2IndexedElementInstruction(TR::InstOpCode::Mnemonic op,
+                             TR::Node *node,
+                             TR::Register *treg,
+                             TR::Register *s1reg,
+                             TR::Register *s2reg,
+                             uint32_t index, TR::CodeGenerator *cg)
+      : ARM64Trg1Src2Instruction(op, node, treg, s1reg, s2reg, cg), _index(index)
+      {
+      }
+
+   /*
+    * @brief Constructor
+    * @param[in] op : instruction opcode
+    * @param[in] node : node
+    * @param[in] treg : target register
+    * @param[in] s1reg : source register 1
+    * @param[in] s2reg : source register 2
+    * @param[in] extendType : extend type
+    * @param[in] shiftAmount : shift amount
```

Comment:
This should be `index` instead of `shiftAmount`.

Reply:
Thanks for pointing it out. Fixed it.